addons/packages/kapp-controller/0.33.1/bundle/config/values.yaml

```diff
@@ -2,6 +2,8 @@
 #@overlay/match-child-defaults missing_ok=True
 ---
 namespace: kapp-controller
+infraProvider: vsphere
```

Comment:
Since this is a new field for kapp-controller, lets make it optional and nullable because I don't want upstream to default to vsphere. The overlay should check if the field is not null.